test/test_quantization.py

```diff
@@ -379,6 +383,29 @@ def test_fuse_module_eval(self):
         self.assertEqual(type(testMod.sub2.bn), torch.nn.BatchNorm2d,
                          "Non-fused submodule BN")
 
+class ObserverTest(QuantizationTestCase):
+    @given(qdtype=st.sampled_from((torch.qint8, torch.quint8)),
```

Comment:
`hypothesis_utils.py` has a tuple with all the quantized types: `_ALL_QINT_TYPES`. You can import it instead of putting them here.

Reply:
I need to test only over qint8 and quint8, qint32 is not a valid choice for calculateQparams.